src/components/metrics/card/name/index.js

```diff
@@ -1,108 +1,117 @@
 import React, {Component, PropTypes} from 'react'
 
-import TextArea from 'react-textarea-autosize'
+import {EditorState, Editor, ContentState, getDefaultKeyBinding, KeyBindingUtil} from 'draft-js'
 
 import './style.css'
 
+class SimpleEditor extends React.Component {
+  state = {
+    editorState: this._plainTextEditorState(this.props.value)
+  }
+
+  _plainTextEditorState(value) {
+    return EditorState.createWithContent(ContentState.createFromText(value || ''))
+  }
+
+  _onChange(editorState) {
+   return this.setState({editorState})
+  }
+
+  focus() {
+    this.refs.editor.focus()
+  }
+
+  changePlainText(value) {
+    this.setState({editorState: this._plainTextEditorState(value)})
```

Comment:
They're similar but different.  Here we wanted some way that the text could be overridden completely from the parent element.  There we wanted to return the planText on every key stroke.  

We could refactor them to make them more similar or the same thing, but I expected to make them even more different later on.